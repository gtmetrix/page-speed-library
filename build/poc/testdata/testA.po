msgid "string_no_original"
msgstr "string_no_original translated"

msgid "string2"
msgstr "string2 translated"

msgid "string1"
msgstr "string1 translated"
