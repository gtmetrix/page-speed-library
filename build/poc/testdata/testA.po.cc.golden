// Generated by poc

// Copyright 2010 Google Inc. All Rights Reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
// Author: aoates@google.com (Andrew Oates)

#include "pagespeed/l10n/register_locale.h"

namespace {

const char* testA_string_table[] = {
  "string1 translated", // string1
  "string2 translated", // string2
  "", // string_no_translations
  0x0
};

pagespeed::l10n::RegisterLocale
  testA_locale("testA", testA_string_table);

} // namespace
